common/resources.yaml

```diff
@@ -81,10 +81,10 @@ groups:
     doc: A table for identifying which objects use which keys.
     dtype:
     - name: objects_idx
-      dtype: uint
+      dtype: int64
```

Comment:
I am surprised that this change is needed. `uint` would be preferred because these indices really should not be negative. Can you try rolling this change back and see what fails?

Reply:
This change back to uint breaks the roundtrip test. @rly